packages/components/src/internal/components/editable/models.test.ts

```diff
@@ -782,26 +782,173 @@ describe('EditorModel', () => {
+                expect(updatedRows[0][expInputCol.fieldKey.toLowerCase()]).toEqual('Value 123, Value 321');
+            });
+
+            test('with original single value, new single value', () => {
+                let updatedRows = emSingleInputs.getUpdatedData(originalSingleValueWithComma);
+                expect(updatedRows[0][expInputCol.fieldKey.toLowerCase()]).toBeUndefined();
+
+                const edWithOriginal = modifyEm(
+                    {
+                        originalData: EditorModel.convertQueryDataToEditorData(originalSingleValueWithComma),
+                    },
+                    emSingleInputs
+                );
+                updatedRows = edWithOriginal.getUpdatedData();
+                expect(updatedRows[0][expInputCol.fieldKey.toLowerCase()]).toBeUndefined();
+
+                updatedRows = emSingleInputs.getUpdatedData(
+                    fromJS({
+                        0: {
+                            [expInputCol.fieldKey]: {
+                                value: 155,
+                                displayValue: 'Not Comma',
+                            },
+                        },
+                        1: {
+                            [expInputCol.fieldKey]: null,
+                        },
+                    })
+                );
+                expect(updatedRows[0][expInputCol.fieldKey.toLowerCase()]).toEqual('"Value 123, Value 321"');
+            });
+
+            test('with original single value, multiple new values', () => {
+                let updatedRows = emMultipleInputs.getUpdatedData(originalSingleValueWithComma);
+                expect(updatedRows[0][expInputCol.fieldKey.toLowerCase()]).toEqual('Value 123, Value 321');
+
+                const edWithOriginal = modifyEm(
+                    {
+                        originalData: EditorModel.convertQueryDataToEditorData(originalSingleValueWithComma),
+                    },
+                    emMultipleInputs
+                );
+                updatedRows = edWithOriginal.getUpdatedData();
+                expect(updatedRows[0][expInputCol.fieldKey.toLowerCase()]).toEqual('Value 123, Value 321');
+            });
+
+            test('with original multi values, single new value', () => {
+                let updatedRows = emSingleInputs.getUpdatedData(originalMultiValues);
+                expect(updatedRows[0][expInputCol.fieldKey.toLowerCase()]).toEqual('"Value 123, Value 321"');
+
+                const edWithOriginal = modifyEm(
+                    {
+                        originalData: EditorModel.convertQueryDataToEditorData(originalMultiValues),
+                    },
+                    emSingleInputs
+                );
+                updatedRows = edWithOriginal.getUpdatedData();
+                expect(updatedRows[0][expInputCol.fieldKey.toLowerCase()]).toEqual('"Value 123, Value 321"');
+            });
+
+            test('with original multi values, multiple new value', () => {
```

Comment:
Can we, and should we, add a case that has multiple values, with one or more of those values having commas in them?

Reply:
good idea. test cases added.